azure-pipelines.yml

```diff
@@ -534,7 +547,7 @@ stages:
         value: $(AutoTestClientSecret_Authed)
       - name: "ESSApiConfiguration.EssClientId"
         value: $(ESSClientId)
-    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'),startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')))
+#    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'),startsWith(variables['Build.SourceBranch'], 'refs/heads/release/')))
```

Comment:
Can I just confirm this condition is supposed to be removed for the QA deployment stage?

Reply:
This condition was commented to verift the changes for QA env. Uncommented now.